app/models/reservation.rb

```diff
@@ -4,8 +4,16 @@ class Reservation < ActiveRecord::Base
 
   scope :unreceived, -> { without_state(:received) }
   scope :received, -> { with_state(:received) }
+  scope :reserved, -> { with_state(:reserved)}
 
   state_machine :initial => :reserved do
+    after_transition any => :shipped do |reservation|
+      reservation.send_purchased_email
+      #send_ship_confirmation_email
```

Comment:
delete the line, add it back when the code supports it, commenting out code clutters the code base